package.json

```diff
@@ -64,6 +65,7 @@
     "@types/node": "~6.0.60",
     "codelyzer": "^4.1.0",
     "concurrently": "^3.5.0",
+    "husky": "^0.14.3",
```

Comment:
do we also need this?

```
  "precommit": ["precommit"],
```

Reply:
it looks like pretty-quick is using husky according to the documentation. looking to see if i can find any more information about that

Reply:
> Husky can prevent bad git commit, git push and more

this is from the husky repo...
i could be wrong but it sounds like husky purpose is to prevent you from committing if your precommit fails. 

Honestly i was just following the docs on prettier site and installed the things they said to install.

I can remove if wanted

Reply:
i think we are good. i tested this elsewhere and here is hwat I have:

```
npm i husky prettier pretty-quick -D
```

Then i added this to the package.json, inside of `scripts`

```
    "pretty-quick": "pretty-quick",
    "precommit": "pretty-quick --staged"
```

The first one is for formatting the entire project, the latter for the hook that husky calls